### ISSUE TITLE
Improving the front page

### ISSUE THREAD

**Issue by bitemyapp** [haskell-lang/haskell-lang#88]
Sharing feedback from a beginner to Haskell:
### Initial feedback

> From the perspective of a newb, https://haskell.org seems to have a helpful landing page while https://haskell-lang.org has a better "Getting Started" section. (edited)
> https://www.haskell.org/downloads is certainly not helpful at all (edited)
> I also wouldn't have noticed the two sites if it weren't for this discussion
### explaining why they preferred haskell.org's front page

> The try-it section, and the part with snippets mostly
> Rather than describing things it shows a beginner how it actually looks
> haskell-lang looks more noisy with the text, it would be nice to have descriptions mixed with code and include the try-it part
> Videos don't matter much to me, but might to some
### My thoughts

I think haskell.org's front page has a bit more flash at the moment, it's possible the designer I retained to do that work exceeded his mandate 😛

**Comment by snoyberg** [comment]
Definitely happy to get some new user feedback on it. I threw out some initial content ideas to @chrisdone for the homepage in an attempt to improve, and hearing how others perceive it is very useful.

**Comment by cloudhead** [comment]
There's definitely a 'wall of text' feeling. Compare to https://www.rust-lang.org/ and https://golang.org/ which don't require any scrolling and have the most important stuff right there.

**Comment by dpwiz** [comment]
Header links really starve for attention due to massive text section below. They need either a hero block for themselves or more whitespace or graphic "breathers" should be added to text sections.

**Comment by bitemyapp** [comment]
I agree with wiz's point about the header links not being very prominent.

Perhaps the content on the front page could get transmogrified into a "Why Haskell?" content page?

**Comment by chrisdone** [comment]
Regarding the Try section, it's just disabled here: https://github.com/haskell-lang/haskell-lang/blob/master/src/HL/View/Home.hs#L28

You can remove the `when False` to get it back, but you'll have to decide where to put it. 👍

**Comment by snoyberg** [comment]
Does someone in this thread want to take a stab at mocking up a replacement homepage? There are lots of great ideas floating around.

**Comment by bitemyapp** [comment]
I could give it a whirl, possibly work with my designer, but I don't know that I'd take the mention of "Try Haskell" on haskell.org as a mandate. I'm a bit squicked out by the limitations of the REPL relative to GHCi and I'd need to test it with some beginners to get a feel for how it comes off.

**Comment by cloudhead** [comment]
I honestly think the "Try X" on language pages are a bit of a gimmick, and a contrived way to try the language anyway. It would be nice to have more code examples though, to flip through.
